.travis.yml

```diff
@@ -30,14 +30,18 @@ install:
   - pip install -U pip
   - conda install --yes -c jjhelmus glpk
   - conda install --yes pigz
-  - conda install --yes -c bioconda diamond=0.8 bowtie2 fastqc=0.11.5 samtools bedtools
+  - conda install --yes -c bioconda diamond=0.8 bowtie2 fastqc=0.11.5 samtools bedtools seqtk
+  - conda install --yes -c knights-lab shogun
   - pip install sphinx sphinx-bootstrap-theme coveralls biopython
   - pip install https://github.com/qiita-spots/qiita_client/archive/master.zip
   - pip install atropos
+  - pip install git+https://github.com/knights-lab/SHOGUN.git
```

Comment:
Is this line really necessary? I thought the conda install `conda install --yes -c knights-lab shogun` will take care of this.

Reply:
Yeah that's shouldn't have been there. It was form an old draft.